### DISCUSSION TITLE
Training time

### DISCUSSION THREAD

**Discussion by dryang21** [mobaidoctor/med-ddpm#9] [General]
May I ask how long it took to train the Mode 1 model? I tried to re-train the model on 40 CT images without modifying the hyperparameters, and it seems to take about 100days to run 100,000  train_num_steps with a workstation equipped with 4 Quadro RTX 8000.

**Comment by mobaidoctor** [comment]
Thank you for reaching out. I've given a detailed response at this link: https://github.com/mobaidoctor/med-ddpm/issues/10#issuecomment-1909409251. Hope it's helpful.
